test/tools/cli/IonProcess.java

```diff
@@ -474,7 +436,7 @@ private static Event ionStreamToEvent(IonReader ionReader) {
                     break;
                 case "field_name":
                     ionReader.stepIn();
-                    String fieldText = null;
+                    String fieldText = "";
```

Comment:
As discussed in a different comment, you should be able to create a SymbolToken with `null` text. This is actually the correct thing to do, since an empty string is a valid field name.

Reply:
Changed, it performances as ion-c now.